app/redux/store.js

```diff
@@ -76,6 +89,7 @@ const zmqMiddlewareCreator = (zmq, subzmq) => {
       zmq.on('message', reply => {
         const data = MsgPack.unpack(reply);
         console.log('Received reply in middleware : ', data);
+        clearTimeout(connectTimer);
```

Comment:
Will the timeout be cleared every single time a reply is received, even though `connectTimer` will b set just once when trying to connect?